sink/datadog.go

```diff
@@ -279,23 +285,29 @@ func (s *Datadog) Send(ctx context.Context, m *blip.Metrics) error {
 			// Convert Blip metric type to Datadog metric type
 			switch metrics[i].Type {
 			case blip.COUNTER:
-				// calculate delta
-				val, ok := s.counters[name]
-				if !ok {
-					blip.Debug("error calculating delta for the counter: %s, no previous value found, skipping this run (next run will work)", name)
-					s.counters[name] = metrics[i].Value
-					continue METRICS
-				}
-				// previous value present, calculate delta
-				delta := metrics[i].Value - val
-				s.counters[name] = metrics[i].Value
-				if delta < 0 {
-					blip.Debug("error calculating delta for the counter: %s, got negative delta (can happen due to restart), skipping this run (next run will work).", name)
-					continue METRICS
+				metricVal := metrics[i].Value
+
+				if s.sendCounterDelta && metrics[i].ValueType != blip.DELTA {
```

Comment:
I propose we change this to 
```suggestion
				if s.sendCounterDelta && metrics[i].ValueType == blip.CUMULATIVE {
```

I'd lean in favor of leaving `UKNOWN` value unchanged.

Reply:
Will update, I was trying to avoid adding `CUMULATIVE` to all other call sites. But you are right, we shouldn't modify `UNKNOWN` values.